store/tikv/lock_resolver.go

```diff
@@ -146,6 +149,79 @@ func (lr *LockResolver) getResolved(txnID uint64) (TxnStatus, bool) {
 	return s, ok
 }
 
+// BatchResolveLocks resolve locks in a batch
+func (lr *LockResolver) BatchResolveLocks(bo *Backoffer, locks []*Lock, loc RegionVerID) (bool, error) {
+	if len(locks) == 0 {
+		return true, nil
+	}
+
+	lockResolverCounter.WithLabelValues("resolve").Inc()
+
+	var expiredLocks []*Lock
+	for _, l := range locks {
+		if lr.store.GetOracle().IsExpired(l.TxnID, l.TTL) {
+			lockResolverCounter.WithLabelValues("expired").Inc()
+			expiredLocks = append(expiredLocks, l)
+		} else {
+			lockResolverCounter.WithLabelValues("not_expired").Inc()
+		}
+	}
+	if len(expiredLocks) == 0 {
+		log.Error("This should not occur!")
+		return true, nil
```

Comment:
True means works have been done, I think we should panic here if it is banned.

Reply:
I think it should return `false` here to indicate there are locks remain to resolve.
Another option is simply skip the expire check, if `BatchResolveLocks` is only designed for GCworker to use, in which case all locks should be expired.

Reply:
i will check whether the count of `lock` and `expiredLocks`, if not, return `false`